### ISSUE TITLE
`Retrieving notices` step fails after python up/downgrade in base environment

### ISSUE THREAD

**Issue by hochreitercorpuls** [conda/conda#11871]
### Checklist

- [X] I added a descriptive title
- [X] I searched open reports and couldn't find a duplicate

### What happened?

If you upgrade or downgrade your python version within your base environment the `Retrieving notices` step fails.
This step was introduced in `conda 4.14.0`

It can easily be reproduced using docker:
```shell
$ docker run --rm -it continuumio/miniconda3:latest
$$ conda update --all # this will install conda 4.14.0
$$ conda install python=3.7 # any version except the used one will work.
...
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
Retrieving notices: ...working... failed
Traceback (most recent call last):
  File "/opt/conda/lib/python3.9/site-packages/conda/exceptions.py", line 1125, in __call__
  File "/opt/conda/lib/python3.9/site-packages/conda/cli/main.py", line 86, in main_subshell
  File "/opt/conda/lib/python3.9/site-packages/conda/cli/conda_argparse.py", line 93, in do_call
  File "/opt/conda/lib/python3.9/site-packages/conda/notices/core.py", line 75, in wrapper
  File "/opt/conda/lib/python3.9/site-packages/conda/notices/core.py", line 39, in display_notices
  File "/opt/conda/lib/python3.9/site-packages/conda/notices/http.py", line 36, in get_notice_responses
  File "/opt/conda/lib/python3.9/site-packages/conda/notices/http.py", line 39, in <genexpr>
  File "/opt/conda/lib/python3.9/concurrent/futures/_base.py", line 609, in result_iterator
  File "/opt/conda/lib/python3.9/concurrent/futures/_base.py", line 446, in result
  File "/opt/conda/lib/python3.9/concurrent/futures/_base.py", line 391, in __get_result
  File "/opt/conda/lib/python3.9/concurrent/futures/thread.py", line 58, in run
  File "/opt/conda/lib/python3.9/site-packages/conda/notices/http.py", line 42, in <lambda>
  File "/opt/conda/lib/python3.9/site-packages/conda/notices/cache.py", line 37, in wrapper
  File "/opt/conda/lib/python3.9/site-packages/conda/notices/http.py", line 58, in get_channel_notice_response
  File "/opt/conda/lib/python3.9/site-packages/requests/sessions.py", line 600, in get
  File "/opt/conda/lib/python3.9/site-packages/requests/sessions.py", line 587, in request
  File "/opt/conda/lib/python3.9/site-packages/requests/sessions.py", line 701, in send
  File "/opt/conda/lib/python3.9/site-packages/requests/adapters.py", line 460, in send
  File "/opt/conda/lib/python3.9/site-packages/requests/adapters.py", line 263, in cert_verify
OSError: Could not find a suitable TLS CA certificate bundle, invalid path: /opt/conda/lib/python3.9/site-packages/certifi/cacert.pem

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/bin/conda", line 13, in <module>
    sys.exit(main())
  File "/opt/conda/lib/python3.9/site-packages/conda/cli/main.py", line 129, in main
  File "/opt/conda/lib/python3.9/site-packages/conda/exceptions.py", line 1413, in conda_exception_handler
  File "/opt/conda/lib/python3.9/site-packages/conda/exceptions.py", line 1128, in __call__
  File "/opt/conda/lib/python3.9/site-packages/conda/exceptions.py", line 1170, in handle_exception
  File "/opt/conda/lib/python3.9/site-packages/conda/exceptions.py", line 1181, in handle_unexpected_exception
  File "/opt/conda/lib/python3.9/site-packages/conda/exceptions.py", line 1251, in print_unexpected_error_report
ModuleNotFoundError: No module named 'conda.cli.main_info'

```

Error message suggests that in the `Retrieving notices` step still the old python environment gets used. (Paths in the Exception message suggest so). As the `Executing transaction` step succeeds prior to that some reactivation of the environment should happen in between.  
However the environment is intact at the end. This error prohibits the downgrade in a CI pipeline starting with `conda 4.14.0`.

### Conda Details

<details>
<summary><code>conda info</code></summary>

```shell
(base) root@7c5c1002d0b5:/# conda info

     active environment : base
    active env location : /opt/conda
            shell level : 1
       user config file : /root/.condarc
 populated config files :
          conda version : 4.14.0
    conda-build version : not installed
         python version : 3.9.13.final.0
       virtual packages : __linux=5.10.102.1=0
                          __glibc=2.31=0
                          __unix=0=0
                          __archspec=1=x86_64
       base environment : /opt/conda  (writable)
      conda av data dir : /opt/conda/etc/conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /opt/conda/pkgs
                          /root/.conda/pkgs
       envs directories : /opt/conda/envs
                          /root/.conda/envs
               platform : linux-64
             user-agent : conda/4.14.0 requests/2.28.1 CPython/3.9.13 Linux/5.10.102.1-microsoft-standard-WSL2 debian/11 glibc/2.31
                UID:GID : 0:0
             netrc file : None
           offline mode : False
```

</details>
<details>
<summary><code>conda list</code></summary>

```shell
(base) root@7c5c1002d0b5:/# conda list --show-channel-urls
# packages in environment at /opt/conda:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main    defaults
_openmp_mutex             5.1                       1_gnu    defaults
brotlipy                  0.7.0           py39h27cfd23_1003    defaults
ca-certificates           2022.07.19           h06a4308_0    defaults
certifi                   2022.9.14        py39h06a4308_0    defaults
cffi                      1.15.1           py39h74dc2b5_0    defaults
charset-normalizer        2.0.4              pyhd3eb1b0_0    defaults
conda                     4.14.0           py39h06a4308_0    defaults
conda-package-handling    1.9.0            py39h5eee18b_0    defaults
cryptography              37.0.1           py39h9ce1e76_0    defaults
cytoolz                   0.11.0           py39h27cfd23_0    defaults
idna                      3.3                pyhd3eb1b0_0    defaults
ld_impl_linux-64          2.38                 h1181459_1    defaults
libffi                    3.3                  he6710b0_2    defaults
libgcc-ng                 11.2.0               h1234567_1    defaults
libgomp                   11.2.0               h1234567_1    defaults
libstdcxx-ng              11.2.0               h1234567_1    defaults
ncurses                   6.3                  h5eee18b_3    defaults
openssl                   1.1.1q               h7f8727e_0    defaults
pip                       22.1.2           py39h06a4308_0    defaults
pycosat                   0.6.3            py39h27cfd23_0    defaults
pycparser                 2.21               pyhd3eb1b0_0    defaults
pyopenssl                 22.0.0             pyhd3eb1b0_0    defaults
pysocks                   1.7.1            py39h06a4308_0    defaults
python                    3.9.13               haa1d7c7_1    defaults
readline                  8.1.2                h7f8727e_1    defaults
requests                  2.28.1           py39h06a4308_0    defaults
ruamel_yaml               0.15.100         py39h27cfd23_0    defaults
setuptools                63.4.1           py39h06a4308_0    defaults
sqlite                    3.39.2               h5082296_0    defaults
tk                        8.6.12               h1ccaba5_0    defaults
toolz                     0.11.2             pyhd3eb1b0_0    defaults
tqdm                      4.64.0           py39h06a4308_0    defaults
tzdata                    2022c                h04d1e81_0    defaults
urllib3                   1.26.11          py39h06a4308_0    defaults
wheel                     0.37.1             pyhd3eb1b0_0    defaults
xz                        5.2.5                h7f8727e_1    defaults
yaml                      0.2.5                h7b6447c_0    defaults
zlib                      1.2.12               h5eee18b_3    defaults
```

</details>
<details>
<summary><code>docker image</code></summary>

```shell
PS C:\Users> docker image inspect continuumio/miniconda3:latest
[
    {
        "Id": "sha256:ce7d119281a1f4685ce6ca66b355c88baa44522ac6a54aee86be96d14ab6dfda",
        "RepoTags": [
            "continuumio/miniconda3:latest"
        ],
        "RepoDigests": [
            "continuumio/miniconda3@sha256:977263e8d1e476972fddab1c75fe050dd3cd17626390e874448bd92721fd659b"
        ],
        "Parent": "",
        "Comment": "buildkit.dockerfile.v0",
        "Created": "2022-06-13T15:29:22.578192309Z",
        "Container": "",
        "ContainerConfig": {
            "Hostname": "",
            "Domainname": "",
            "User": "",
            "AttachStdin": false,
            "AttachStdout": false,
            "AttachStderr": false,
            "Tty": false,
            "OpenStdin": false,
            "StdinOnce": false,
            "Env": null,
            "Cmd": null,
            "Image": "",
            "Volumes": null,
            "WorkingDir": "",
            "Entrypoint": null,
            "OnBuild": null,
            "Labels": null
        },
        "DockerVersion": "",
        "Author": "",
        "Config": {
            "Hostname": "",
            "Domainname": "",
            "User": "",
            "AttachStdin": false,
            "AttachStdout": false,
            "AttachStderr": false,
            "Tty": false,
            "OpenStdin": false,
            "StdinOnce": false,
            "Env": [
                "PATH=/opt/conda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
                "LANG=C.UTF-8",
                "LC_ALL=C.UTF-8"
            ],
            "Cmd": [
                "/bin/bash"
            ],
            "ArgsEscaped": true,
            "Image": "",
            "Volumes": null,
            "WorkingDir": "",
            "Entrypoint": null,
            "OnBuild": null,
            "Labels": {
                "maintainer": "Anaconda, Inc"
            }
        },
        "Architecture": "amd64",
        "Os": "linux",
        "Size": 403452954,
        "VirtualSize": 403452954,
        "GraphDriver": {
            "Data": {
                "LowerDir": "/var/lib/docker/overlay2/1836289c9668568f0b2d7661e68448905f4b02253d5551fbf6695cb9507264d8/diff:/var/lib/docker/overlay2/38e39804e130bbd88624f533612a66747a7cc92931ed9b76209a605d4d9a273f/diff",
                "MergedDir": "/var/lib/docker/overlay2/b2e6cf2c26ce97f8fdb8d1f8b60a6bfc586c8897090340330b59f06c18df84ec/merged",
                "UpperDir": "/var/lib/docker/overlay2/b2e6cf2c26ce97f8fdb8d1f8b60a6bfc586c8897090340330b59f06c18df84ec/diff",
                "WorkDir": "/var/lib/docker/overlay2/b2e6cf2c26ce97f8fdb8d1f8b60a6bfc586c8897090340330b59f06c18df84ec/work"
            },
            "Name": "overlay2"
        },
        "RootFS": {
            "Type": "layers",
            "Layers": [
                "sha256:ad6562704f3759fb50f0d3de5f80a38f65a85e709b77fd24491253990f30b6be",
                "sha256:6fa1f4185aa209b8bfd3075ccbdb3cca377213a724a98361c0bc0f1f63013836",
                "sha256:ab2731ec3f53b2bfa28594ee8b580eb4fb9e55c3e3e3c72c675f84afa7fa4937"
            ]
        },
        "Metadata": {
            "LastTagTime": "0001-01-01T00:00:00Z"
        }
    }
]
```

</details>

**Comment by kenodegard** [comment]
@hochreitercorpuls thanks for reporting this error, this is a duplicate of #11847 and a fix is already in progress #11852

**Comment by hochreitercorpuls** [comment]
@kenodegard - sorry I had not found the other issue.
